### PR TITLE
feat: upgrade liquidjs to ^9.28.4, fixes #1995

### LINKED ISSUE #1995: liquidjs Regression on Relative Includes
Related to https://github.com/harttle/liquidjs/issues/395#issuecomment-933007299

Locked to `liquidjs` 9.25.x starting in Beta 1.

cc @paulrobertlloyd

### PR DESCRIPTION
**Pull request by harttle**
Fixes #1995

- fix failing test cases, mostly by adding the resolved file into `includes`.
- restore all `.skip`ed cases, clarify LiquidJS usage.

### CODE REVIEW

File: test/TemplateRenderLiquidTest.js [hunk 1]
```diff
@@ -102,6 +103,11 @@ test("Liquid Render Relative (current dir) Include", async (t) => {
       liquidOptions: {
         dynamicPartials: false,
       },
+    },
+    {
+      dir: {
+        includes: "relative-liquid",
```
Comment (harttle) [inline test/TemplateRenderLiquidTest.js:109]: resolved fullpath will be ignored if not within `includes` directory

File: test/TemplateRenderLiquidTest.js [hunk 2]
```diff
@@ -117,17 +123,28 @@ test("Liquid Render Relative (parent dir) Include", async (t) => {
...
-test.skip("Liquid Render Relative (relative include should ignore _includes dir) Include", async (t) => {
+test("Liquid Render Relative (relative include should ignore _includes dir) Include", async (t) => {
   let tr = getNewTemplateRender(
     "./test/stubs/does_not_exist_and_thats_ok.liquid",
-    "./test/stubs/"
+    "./test/stubs/",
+    {},
+    {
+      dir: {
+        includes: ".",
```
Comment (harttle) [inline test/TemplateRenderLiquidTest.js:145]: filepath relative to current file has higher priority than filepath relative to root directory, but still it need to be within `includes`

File: test/TemplateRenderLiquidTest.js [hunk 3]
```diff
@@ -704,17 +733,32 @@ test("Liquid Shortcode Multiple Args", async (t) => {
...
+  // see: https://github.com/harttle/liquidjs/issues/404#issuecomment-955660149
   let fn = await getNewTemplateRender(
     "liquid",
     "./test/stubs/"
-  ).getCompiledTemplate("<p>{% include scopeleak %}{{ test }}</p>");
+  ).getCompiledTemplate("<p>{% include 'scopeleak' %}{{ test }}</p>");
+  t.is(await fn({ test: 1 }), "<p>22</p>");
+});
+
+test("Liquid Render Scope Leak", async (t) => {
```
Comment (harttle) [inline test/TemplateRenderLiquidTest.js:752]: add a test for `{%render%}` to demonstrate the difference between `render` and legacy `include`.

File: test/TemplateRenderLiquidTest.js [hunk 4]
```diff
@@ -957,15 +1001,17 @@ test("Issue 600: Liquid Shortcode argument with underscores", async (t) => {
   );
 });
 
-test.skip("Issue 611: Run a function", async (t) => {
-  // This works in Nunjucks
+test("Issue 611: Run a function", async (t) => {
+  // function calls in Nunjucks can be replaced by custom Drops
   let tr = getNewTemplateRender("liquid", "./test/stubs/");
-
+  class CustomDrop extends Drop {
```
Comment (harttle) [inline test/TemplateRenderLiquidTest.js:1007]: is this useful at all? Liquid use a concept `Drop` to implement scenarios like this, even has a ruby like `liquidMethodMissing()` method available.

**Comment by zachleat** [comment]
Wow. This is incredible—thank you! Those skipped tests were unrelated to #1995, for the record—so this is above and beyond! 🏆🏆🏆🏆

File: package.json [hunk 5]
```diff
@@ -80,6 +80,7 @@
     "husky": "^7.0.4",
     "js-yaml": "^4.1.0",
     "lint-staged": "^11.2.6",
+    "liquidjs": "^9.28.5",
```
Comment (pdehaan) [inline package.json:83]: I think **liquidjs** might need to move back into the `dependencies` [vs `devDependencies`].

Comment (zachleat) [reply package.json:83]: +1 I’ll do that post-merge

File: test/TemplateRenderLiquidTest.js (hunk 4 above)
Comment (zachleat) [reply test/TemplateRenderLiquidTest.js:1007]: Yeah, I like this!

**Comment by zachleat** [comment]
Made a few changes locally to avoid adding template config to those tests, but this was great—thank you!

**Pull request merged**